modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -397,6 +387,74 @@ private void listenForTableChange() {
 
     /** {@inheritDoc} */
     @Override public Table table(String name) {
```

Comment:
Is that true that currently table is created within specific schema scope, PUBIC bu default?
If true, seems that it should be possible to have tables with same name in different schemas, is that correct?
If true, how it's possible to get tables with same name from different schemas?

Reply:
Now we are supporting only canonical name here, in other word the table would be return if and only if you pass its schema and name ({SCHEMA}.{NAME}).